src/Workspaces/Core/Portable/Rename/Renamer.cs

```diff
@@ -75,6 +76,12 @@ public static partial class Renamer
                 throw new ArgumentNullException(nameof(document));
             }
 
+            if (document.Services.GetService<ISpanMappingService>() != null)
```

Comment:
Can we add a test for this in [RenamerTests](http://sourceroslyn.io/#Microsoft.CodeAnalysis.Workspaces.UnitTests/Renamer/RenamerTests.cs,d3fdb62c67de81fe)?